### ISSUE TITLE
[Bug] Can't move Player/Unit/target frames

### ISSUE THREAD

**Issue by maniak262** [Krealle/Cell_UnitFrames#77]
### Addon Version

1.3.4

### World of Warcraft Version

Retail

### Describe the bug

I cannot remove any of the Unit frames.  I am sorry if its a dumb question.  I've looked all over for a option and tried different modifiers on the window.

### Steps to reproduce

.

### BugSack

```bash
.
```

**Comment by Krealle** [comment]
You press this button to open up `Edit Mode`.
I'm workshopping some changes to make it more apparent.
![image](https://github.com/user-attachments/assets/c4355a37-726f-4965-b89d-357d628e8240)

**Comment by maniak262** [comment]
First thank you.  I looked everywhere and it was right in front of me. My suggestion is rename it to :Unlock Unit frames anchor" and change the color to red for it to stand out.  Everything else was clear on my end.  I attached Mua portraits to mine via weak aura.  Not a big deal but since the target portrait is on the right the a "Reverse" grow direction on health/mana would be handy but no big.  Appreciate this and love the addon.  I loved cell but wanted it to just replace them all.
